les-4-opdracht-transition/styles.css

```diff
@@ -3,12 +3,23 @@
     --menu-bar-height: 10px;
```

Comment:
Wat goed van jou dat je ook gebruik gemaakt hebt van een variabele. Minder code en makkelijk op 1 plek aanpasbaar. 
in de les had Nova dit voorgedaan met de pseudoclass :root, ik weet niet of dat noodzakelijk is.